/**:
  ros__parameters:
    n_channels: 16
    src: 'plughw:1,0'
    sample_rate: 44100
    hop_size: 44100
    frame_size: 44100 # publish 1 second every second
    microphone_frame_id: 'philbart/base_link'
